synapse/models/person.py

```diff
@@ -22,24 +22,22 @@ def getModelDefs(self):
                 }),
                 ('ps:name', ('str', {'lower': True, 'onespace': True}), {
                     'doc': 'An arbitrary, lower spaced string with normalized whitespace.',
-                    'ex': 'robert grey'
-                }),
+                    'ex': 'robert grey'}),
```

Comment:
I keep seeing "Bob Grey" in the code/tests. Is this some pop reference I'm missing?

Reply:
several i suspect ;)